components/collator/benches/bench.rs

```diff
@@ -0,0 +1,89 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+use std::time::Duration;
+
+use criterion::{criterion_group, criterion_main, BatchSize, BenchmarkId, Criterion};
+
+use icu::collator::*;
+use icu::locid::Locale;
+
+pub fn collator_with_locale(criterion: &mut Criterion) {
+    // Load file content in reverse order vector.
+    let content_latin: (&str, Vec<&str>) = (
+        "TestNames_Latin",
+        include_str!("data/TestNames_Latin.txt")
+            .split('\n')
+            .rev()
+            .collect::<Vec<&str>>(),
+    );
+    let content_asian: (&str, Vec<&str>) = (
+        "TestNames_Asian",
+        include_str!("data/TestNames_Asian.txt")
+            .split('\n')
+            .rev()
+            .collect(),
+    );
+    let content_russian: (&str, Vec<&str>) = (
+        "TestNames_Russian",
+        include_str!("data/TestNames_Russian.txt")
+            .split('\n')
+            .rev()
+            .collect(),
+    );
+
+    let performance_parameters = [
+        ("en_US", vec![&content_latin]),
+        ("fr_FR", vec![&content_latin]),
+        ("ja_JP", vec![&content_latin, &content_asian]),
+        ("zh_CN", vec![&content_latin]),
+        ("ru_RU", vec![&content_latin, &content_russian]),
+    ];
```

Comment:
I wonder if the `ja-JP` and `ru-RU` cases end up with an unrealistic amount of Latin input mixed with the native-script input with the data files being the way they are.

Reply:
I've updated the set of files, locale and strength under bench. 

Please have a look again !